packages/react-core/src/components/Select/examples/Select.md

```diff
@@ -589,8 +589,7 @@ class FilteringCheckboxSelectInput extends React.Component {
       }
     };
 
-    this.onFilter = evt => {
```

Comment:
These `this.onFilter` methods in the examples also need to be changed to have `textInput` as the second arg now I think

Reply:
Good catch, thanks.